lib/replica.rb

```diff
@@ -155,12 +155,16 @@ def compile_query_url(endpoint, query)
     "#{base_url}#{endpoint}?#{language_and_project_params}&#{query}"
   end
 
-  # We must special-case Multilingual Wikisource because its labs database name
-  # does not follow the normal pattern; it is called 'sourceswiki', which is the
-  # same one we'd get if a Wikipedia's language was "sources".
+  # We must special-case Multilingual Wikisource/ Wikimedia Incubator because
+  # their labs database names do not follow the normal pattern; they are called
+  # 'sourceswiki' and 'incubatorwiki' respectively, which are the same as we'd
+  # get if a Wikipedia's language was "sources" or "incubator". Special case for
+  # Wikimedia however also handles addition of future sub-domains.
   def language_and_project_params
     if @wiki.project == 'wikisource' && @wiki.language.nil?
       'lang=sources&project=wikipedia'
+    elsif @wiki.project == 'wikimedia' && @wiki.language
```

Comment:
I'm not sure how general this is. For now, we should just handle the explicit case of project 'wikimedia', language 'incubator'. here.